firmware/controllers/trigger/trigger_decoder.cpp

```diff
@@ -209,7 +209,7 @@ void TriggerState::decodeTriggerEvent(trigger_event_e const signal, efitime_t no
 	if (isLessImportant(type)) {
 #if EFI_UNIT_TEST || defined(__DOXYGEN__)
 		if (printTriggerDebug) {
-			printf("%s isLessImportant %s now=%d index=%d\r\n",
+			printf("%s isLessImportant %s now=%lld index=%d\r\n",
```

Comment:
nowNt is 64-bit, and if we use %d instead of %lld, then index will be messed up...